FreeRTOS/FreeRTOS-Plus-TCP
# Fix unit tests for ARP file

Description
-----------
This PR fixes the build issues with unit tests for the ARP source file.

Test Steps
-----------
Tests building:
`
cmake -S test/unit-test -B test/unit-test/build/
make -C test/unit-test/build/ all -j6
`

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
